javalib/src/main/scala/java/lang/System.scala

```diff
@@ -23,7 +23,12 @@ object System {
   def identityHashCode(x: Object): scala.Int =
     x.cast[Word].hashCode
 
-  def getenv(name: String): String                      = ???
+  def getenv(name: String): String = {
+    val cResult = stdlib.getenv(toCString(name))
+    fromCString(cResult)
```

Comment:
What happens if the property you're looking up doesn't exist? I think this code will segfault, but in Java you're supposed to return `null`.